docs/index.asciidoc

```diff
@@ -48,9 +50,9 @@ output {
 }
 -----
 <1> `%{[@metadata][beat]}` sets the first part of the index name to the value
-of the `beat` metadata field and `%{[@metadata][version]}` sets the second part to
-the {plugin-uc} version. For example:
-metricbeat-7.4.0.
+of the metadata field and `%{[@metadata][version]}` sets the second part to
+the {plugin-singular} version. For example:
+metricbeat-7.4.0. or elastic_agent-7.14.0.
```

Comment:
@andsel Will this work as documented? We're relying on metadata, so I think it might not.

Reply:
@karenzone we should remove any reference to `[@metadata][beat]` and  `[@metadata][version]`  because the agent doesn't send that information